source/construct-relators.rq

```diff
@@ -1,16 +1,16 @@
 prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
-prefix owl: <http://www.w3.org/2002/07/owl#>
 prefix skos: <http://www.w3.org/2004/02/skos/core#>
 prefix bf2: <http://id.loc.gov/ontologies/bibframe/>
+prefix madsrdf: <http://www.loc.gov/mads/rdf/v1#>
 prefix : <https://id.kb.se/vocab/>
 
 construct {
     ?idrel a bf2:Role;
         skos:notation ?notation;
         ?p ?o;
-        owl:equivalentProperty ?locrel;
+        skos:exactMatch ?locrel;
```

Comment:
We should use `owl:equivalentProperty` (or add both if we need `skos:exactMatch` in our logic) as per the (rocket) discussion.

Reply:
Sure. So which is preferrable, owl:equivalentProperty or both? Is there any particular reason relators should use this when e.g. languages such as https://libris.kb.se/katalogisering/766vj3jc919cchbh uses only exactMatch?

Reply:
I'd say ` owl:equivalentProperty`, but both can be used (for e.g. our display of these, and potentially queries or data consumers in general). (I'd prefer to declare ` kbvl:equivalentProperty` as a subproperty of both of these, but let's do that another day).

The reason is that these relators *are* properties (could be used as direct properties on entities), whereas languages are not.